Make committer applications process lighter (#101)
PrestaShop/open-source: content/maintainers-guide/how-to-become-a-committer.md

> ## Requirements to become a committer

Reviewer: ```suggestion
## Prerequisites to become a committer
```

1. **Active participation in the project**

>     - Have at least one accepted, quality Pull Request

Reviewer: I think this part should be kept, otherwise we'll end up with useless committer applications, or people who will only contribute with useless "LGTM"s
```suggestion
    - Have at least one accepted, quality Pull Request.
    - Display a record of Pull Request reviews.
```

## How to apply to be a committer

1. [Open an issue on the open source repository](https://github.com/PrestaShop/open-source/issues/new) explaining why you apply to become a committer, and on what part of the project (Core or module/subproject).
2. If three maintainers approve the application, the new committer is approved.

> 3. After one week, if the issue did not receive three positive votes, it is considered rejected and closed.

Reviewer: One week might be too short (eg. if people are too busy or on vacation)

Author: 30 days then

> 3. After one week, if the issue did not receive three positive votes, it is considered rejected and closed.

Author: ```suggestion
3. After 30 days, if the issue did not receive three positive votes, it is considered rejected and closed.
```

- Lack of activity for the past three months.
- Willful negligence or intent to harm the PrestaShop project.
- On their demand, for personal reasons.

> - If a simple majority of maintainers are willing to

Reviewer: Lazy consensus
```suggestion
- On request of at least three maintainers.
```
